pdata/internal/wrapper_value.go

```diff
@@ -415,5 +417,137 @@ func MarshalProtoOrigAnyValue(orig *otlpcommon.AnyValue, buf []byte) int {
 }
 
 func UnmarshalProtoOrigAnyValue(orig *otlpcommon.AnyValue, buf []byte) error {
+	var err error
+	var fieldNum int32
+	var wireType proto.WireType
+
+	l := len(buf)
+	pos := 0
+	for pos < l {
+		// If in a group parsing, move to the next tag.
+		fieldNum, wireType, pos, err = proto.ConsumeTag(buf, pos)
+		if err != nil {
+			return err
+		}
+		switch fieldNum {
+		case 1:
```

Comment:
Should we make constants? In case we ever add a new type, it'll be easier to notice

Reply:
Very soon (next week) will automate this as well and will be generate by pdatagen. I would not be worry at this moment.